patches/minecraft/net/minecraft/client/renderer/RenderItem.java.patch

```diff
@@ -9,6 +9,15 @@
          this.func_175041_b();
          this.field_184395_f = p_i46552_3_;
      }
+@@ -147,7 +147,7 @@
+             }
+             else
+             {
+-                this.func_175036_a(p_180454_2_, p_180454_1_);
++                for (List<BakedQuad> quads : net.minecraftforge.client.ForgeHooksClient.sortQuadsByIndex(p_180454_2_)) { net.minecraftforge.client.ForgeHooksClient.renderItemLayerPre(p_180454_1_, quads.get(0).func_178211_c()); this.func_175032_a(Tessellator.func_178181_a().func_178180_c(), quads, -1, p_180454_1_); net.minecraftforge.client.ForgeHooksClient.renderItemLayerPost(p_180454_1_, quads.get(0).func_178211_c()); }
```

Comment:
the rule is not a hard one. minimize sizes where you can but it still needs to be readable.